.github/workflows/upgrade-python-requirements.yml

```diff
@@ -3,20 +3,20 @@ name: Upgrade Requirements
 on:
   schedule:
     # will start the job at 13:15 UTC on 1st of every month
-    - cron: "15 13 1 * *"
+    - cron: "15 13 3 * *"
   workflow_dispatch:
     inputs:
       branch:
         description: "Target branch to create requirements PR against"
         required: true
-        default: 'main'
+        default: '$default-branch'
 
 jobs:
 
   call-upgrade-python-requirements-workflow:
      uses: edx/.github/.github/workflows/upgrade-python-requirements.yml@master
      with:
-       branch: ${{ github.event.inputs.branch }}
+       branch: ${{ github.event.inputs.branch || '$default-branch' }}
```

Comment:
can you explain what is going on here?

Reply:
So here we are using macro $default-branch so it will either get the default branch or it will get the branch which is raised via event 
you can refer this for macro : https://github.blog/changelog/2020-07-22-github-actions-better-support-for-alternative-default-branch-names/